neat-dnfs/analysis/tests/test_stats.py

```
@@ -0,0 +1,171 @@
+    assert _student_t_two_tailed_pvalue(2.228, 10) == pytest.approx(0.05, abs=1e-3)
+    assert _student_t_two_tailed_pvalue(0.0, 10) == pytest.approx(1.0)
+
+
+def test_spearman_correlation_perfect_monotonic_is_plus_one():
+    rho, p = spearman_correlation([1, 2, 3, 4, 5], [10, 20, 30, 40, 50])
+    assert rho == pytest.approx(1.0)
+    assert p == pytest.approx(0.0)
+
+
+def test_spearman_correlation_perfect_inverse_is_minus_one():
+    rho, p = spearman_correlation([1, 2, 3, 4, 5], [50, 40, 30, 20, 10])
```

Ruff (0.16.1)

[warning] Unpacked variable `p` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)